<a href="https://colab.research.google.com/github/cristianramirezrodriguez/practicealgebra1/blob/main/SolutionsOneVariableEquations.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**Solution set for PracticeAssignment OneVariableEquations.ipynb**

For my students from EBIA, this is Tech Worksheet 2 Part B

Name: Cristian Ramirez Rodriguez

Any Collaborator Names:

# We have practiced extensively on solving for x when presented a one-variable equation. Our goal today is to use technology to help our process.

Our first step is to move all terms to the left-hand-side of the equation then inputting the left-hand expression into the place where it says "expression" in our python solver.

# The syntax to find the x value of a given one-variable equation is as follows:

from sympy.solvers import solve

from sympy import Symbol

x = Symbol('x')

solve(expression, x)


##For example

## Say you had the equation $x+3 = 5$

## We will first move all terms to the left hand side, producing $x+3-5=0$.

##We can then input the expression on the left-hand side in the solver and run the code below (which produced the correct answer, x=2).

In [1]:
from sympy.solvers import solve
from sympy import Symbol
x = Symbol('x')
solve(x + 3 - 5, x)

[2]

Another example:

Say you want to find $42.1 - x = 2 + x$

You would first move everything over to the left hand side, producing

$42.1-x-2-x=0$

and then input the lefthand expression: $42.1-x-2-x$ into the solver as seen below:

In [2]:
from sympy.solvers import solve
from sympy import Symbol
x = Symbol('x')
solve(42.1-x-2-x, x)


[20.0500000000000]

This produces 20.05, which is the correct answer.

# For each of the following, find x:

1. $2=7x$

2. $-4 = 6x$

3. $1=x+1$

4. $2 = 2x + 22$

5. $12.77 = 33x -2$

6. $9x = 25.9 - 3$

7. $3543x=43-x$

8. $45x + 44 = -35x +2$

9. $2-5=10(x-4)$

10. $4x+5y=0$

11. $y = mx+b$


In [3]:
#Do number 1 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
sym.Rational(solve(2-7*x, x)[0]) #The added code gives us a fraction in lowest terms
#x is 2/7

2/7

In [4]:
#Do number 2 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
sym.Rational(solve(-4-6*x, x)[0])

-2/3

In [5]:
#Do number 3 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
sym.Rational(solve(1-x-1, x)[0])

0

In [6]:
#Do number 4 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
sym.Rational(solve(2-2*x-22, x)[0])

-10

In [7]:
#Do number 5 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
solve(12.77-33*x+2, x)

[0.447575757575758]

In [8]:
#Do number 6 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
solve(9*x-25.9+3, x)[0]

2.54444444444444

In [9]:
#Do number 7 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
sym.Rational(solve(3543*x-43+x, x)[0])

43/3544

In [10]:
#Do number 8 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
sym.Rational(solve(45*x+44+35*x-2, x)[0])

-21/40

In [11]:
#Do number 9 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
sym.Rational(solve(2-5-10*(x-4), x)[0])

37/10

In [12]:
#Do number 10 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
y = Symbol('y') #with more variables, need to name them
solve(4*x+5*y, x) #Since no longer a number but expression can't use sym.Rational

[-5*y/4]

In [13]:
#Do number 11 here
#Do number 10 here
from sympy.solvers import solve
import sympy as sym
from sympy import Symbol
x = Symbol('x')
y = Symbol('y') #with more variables, need to name them
m = Symbol('m')
b = Symbol('b')
solve(y-m*x-b, x)

[(-b + y)/m]